src/app/personside/infotabs/utbetalinger/utbetaling/EnkelUtbetaling.tsx

```diff
@@ -63,125 +50,110 @@ const UtbetalingHeaderStyle = styled.div`
-                        <UtbetalingTabellStyling>
-                            <UtbetalingHeaderStyle>
-                                <SpaceBetween>
-                                    <Normaltekst tag={'h4'}>
-                                        <Bold>{tittel}</Bold>
-                                    </Normaltekst>
-                                    <Normaltekst>
-                                        <Bold>{sum}</Bold>
-                                    </Normaltekst>
-                                </SpaceBetween>
-                                <Normaltekst className="order-first">
-                                    {dato} / <Bold>{utbetaling.status}</Bold>
-                                </Normaltekst>
-                                <SpaceBetween>
-                                    <Normaltekst>{periode}</Normaltekst>
-                                    <Normaltekst>{forfallsInfo}</Normaltekst>
-                                </SpaceBetween>
-                                <SpaceBetween>
-                                    <Normaltekst>Utbetaling til: {utbetaling.utbetaltTil}</Normaltekst>
-                                    <span ref={this.printerButtonRef}>
-                                        <PrintKnapp onClick={this.handlePrint} />
-                                    </span>
-                                </SpaceBetween>
-                            </UtbetalingHeaderStyle>
-                            <DetaljerCollapse open={this.props.visDetaljer} toggle={this.toggleVisDetaljer}>
-                                <UtbetalingsDetaljer
-                                    ytelse={ytelse}
-                                    konto={utbetaling.konto}
-                                    melding={utbetaling.melding}
-                                />
-                            </DetaljerCollapse>
-                        </UtbetalingTabellStyling>
-                    </article>
-                </UtbetalingStyle>
-            </Printer>
-        );
-    }
-}
-
-function mapDispatchToProps(dispatch: Dispatch<AnyAction>, ownProps: OwnProps): DispatchProps {
-    return {
-        setYtelseIFokus: () => dispatch(setNyYtelseIFokus(ownProps.ytelse)),
-        setEkspanderYtelse: (ekspander: boolean) => dispatch(setEkspanderYtelse(ownProps.ytelse, ekspander))
     };
-}
 
-function mapStateToProps(state: AppState, ownProps: OwnProps): StateProps {
-    return {
-        erIFokus: state.utbetalinger.ytelseIFokus === ownProps.ytelse,
-        visDetaljer: state.utbetalinger.ekspanderteYtelser.includes(ownProps.ytelse)
-    };
+    return (
+        <PrinterWrapper>
+            <UtbetalingStyle
+                onClick={(event: React.MouseEvent<HTMLElement>) => {
+                    cancelIfHighlighting(() => handleClickOnUtbetaling(event));
+                }}
+                ref={utbetalingRef}
+                tabIndex={0}
+                //onFocus={setYtelseIFokus}
```

Comment:
@winsvold denne fjernet jeg for å få onClicken til å fungere. Har du forslag på hvordan vi kan fikse dette på en ryddig måte?

Reply:
Må nesten ta en titt på koden for å forstå hva onFocus gjør her